[2023-04-26T04:24:04.149+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:24:02.139773+00:00 [queued]>
[2023-04-26T04:24:04.156+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:24:02.139773+00:00 [queued]>
[2023-04-26T04:24:04.157+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:24:04.157+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T04:24:04.158+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T04:24:04.167+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-26 04:24:02.139773+00:00
[2023-04-26T04:24:04.173+0000] {standard_task_runner.py:55} INFO - Started process 21737 to run task
[2023-04-26T04:24:04.178+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-26T04:24:02.139773+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpoxyc7ect']
[2023-04-26T04:24:04.181+0000] {standard_task_runner.py:83} INFO - Job 146: Subtask is_api_available
[2023-04-26T04:24:04.234+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-26T04:24:02.139773+00:00 [running]> on host bebb68623a17
[2023-04-26T04:24:04.281+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T04:24:02.139773+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T04:24:02.139773+00:00
[2023-04-26T04:24:04.282+0000] {http.py:120} INFO - Poking: api/
[2023-04-26T04:24:04.290+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-26T04:24:04.291+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-26T04:24:04.931+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-26T04:24:04.943+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230426T042402, start_date=20230426T042404, end_date=20230426T042404
[2023-04-26T04:24:04.968+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T04:24:04.992+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
